src/st/find_entity_horizontal.h

```diff
@@ -11,10 +11,11 @@
  *    `g_LayoutObjHorizontal` will be positioned at an element with a
  *    x position larger than `posX` or the tail eleement.
  */
-void FindFirstEntityToTheRight(s16 posX) {
+static void FindFirstEntityToTheRight(s16 posX) {
```

Comment:
These needed to be marked as static to resolve register swaps on PSP where these functions were called.

This [has been discussed recently on Discord](https://discord.com/channels/1079389589950705684/1079395108501331990/1326286609309630496) and may give us further PSP matches on code which is only just off.